src/rpc/blockchain.cpp

```diff
@@ -2255,6 +2258,7 @@ static RPCHelpMan dumptxoutset()
         "Write the serialized UTXO set to disk.",
         {
             {"path", RPCArg::Type::STR, RPCArg::Optional::NO, "Path to the output file. If relative, will be prefixed by datadir."},
+            {"format", RPCArg::Type::STR, RPCArg::Default{"compact"}, "'compact' to output a compact binary serialized format, 'sqlite' to output a SQLite database with table 'txoutset' (requires sqlite3)", "format"},
```

Comment:
This outputs the following:
```
2. format    (string, optional, default="compact") 'compact' to output a compact binary serialized format, 'sqlite' to output a SQLite database with table 'txoutset' (requires sqlite3)
```
Maybe use quotes the same as the default.
```suggestion
            {"format", RPCArg::Type::STR, RPCArg::Default{"compact"}, "\"compact\" to output a compact binary serialized format, \"sqlite\" to output a SQLite database with table \"txoutset\" (requires sqlite3)", "format"},
```